### PR TITLE
Feature/show form error

### PR DESCRIPTION
**Pull request by AlviseSembenico**

### CODE REVIEW

File: src/vng/templates/forms.html [hunk 1]
```diff
@@ -16,6 +18,7 @@
 
 {% block input %}
     {% with input_type=input_type|default:"text" %}
+    {{n}}
```
Comment (alextreme) [inline src/vng/templates/forms.html]: Not sure you want to show this?

File: src/vng/templates/forms.html [hunk 2]
```diff
@@ -24,15 +27,11 @@
...
                         {{ widget.attrs|flatattrs }}
                         {{ required|yesno:"required," }}
-                        {% if placeholder %}placeholder="{{ placeholder }}"{% endif %}
-
-                >
-                    {% comment %} <div class="invalid-feedback">{{ placeholder }}</div>
-                {% endif %} {% endcomment %}
-                {% reuse '_errors' %}
+                        {% if placeholder %}placeholder="{{ placeholder }}"{% endif %}>
+                {% reuse '_errors' n=n %}
```
Comment (alextreme) [inline src/vng/templates/forms.html]: Ahh nevermind now I understand.

Comment (alextreme) [reply src/vng/templates/forms.html]: maybe make this a bit more explainatory variable name

**Comment by AlviseSembenico** [comment]
@alextreme index as variable name, better? :)

**Comment by alextreme** [comment]
As long as you deal with the remaining {{ n }} :)

**Comment by alextreme** [comment]
@AlviseSembenico and check the conflicts due to the merging of your other branches

**Comment by AlviseSembenico** [comment]
@alextreme fixed

**Comment by alextreme** [comment]
@AlviseSembenico http://jenkins.maykin.nl/job/VNG%20Github/job/api-testvoorziening/view/change-requests/job/PR-156/4/console

**Comment by AlviseSembenico** [comment]
@alextreme fixed

**Comment by AlviseSembenico** [comment]
@alextreme jenkins working http://jenkins.maykin.nl/job/VNG%20Github/job/api-testvoorziening/view/change-requests/job/PR-156/

**Pull request merged**